packages/visual-editor/src/components/Locator.tsx

```
@@ -863,21 +857,30 @@ const LocatorInternal = ({
-            ) {
-              displayName = firstResult.value;
-              centerCoords = [filterFromResult.lng, filterFromResult.lat];
-              return true;
+            const resultFilter = firstResult?.filter;
+            if (!firstResult || !resultFilter) {
+              return false;
+            }
+
+            switch (resultFilter.matcher) {
+              case Matcher.Near:
+                const filterFromResult = resultFilter.value as NearFilterValue;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)